annotation/functions.scm

```diff
@@ -479,25 +491,36 @@
 
   If do-protein is #t then protein interactions are included.
 "
-	(map
-		(lambda (gene-pair)
-			(generate-result (gar gene-pair) (gdr gene-pair) do-protein namespace parents coding non-coding))
-
-		(run-query (Get
-			(VariableList
-				(TypedVariable (Variable "$a") (Type 'GeneNode))
-				(TypedVariable (Variable "$b") (Type 'GeneNode)))
-
-			(And
-				(Evaluation (Predicate "interacts_with")
-					(SetLink gene (Variable "$a")))
+      (map
+         (lambda (gene-pair)
+            (generate-result (gar gene-pair) (gdr gene-pair) do-protein namespace parents coding non-coding))
 
-				(Evaluation (Predicate "interacts_with")
-					(SetLink (Variable "$a") (Variable "$b")))
-
-				(Evaluation (Predicate "interacts_with")
-					(SetLink gene (Variable "$b")))
-			))))
+         (run-query (Get
+            (VariableList
+               (TypedVariable (Variable "$a") (Type 'GeneNode))
+               (TypedVariable (Variable "$b") (Type 'GeneNode)))
+               (And 
+                  (Evaluation (Predicate "interacts_with")
+                  (SetLink gene (Variable "$a")))
+
+                  (Evaluation (Predicate "interacts_with")
+                     (SetLink (Variable "$a") (Variable "$b")))
+
+                  (Evaluation (Predicate "interacts_with")
+                     (SetLink gene (Variable "$b")))
+                  (map (lambda (org)
+                        (Absent 
+                           (Evaluation (Predicate "from_organism")
+                              (List 
+                                 (Variable "$a")
+                                 (ConceptNode (string-append "TaxonomyID:" org))
+                              )
+                           )
+                        )
+                     ) orgs)
```

Comment:
It doesn't really matter all that much, but it becomes slightly easier to understand if this `orgs` here was `exclude-orgs`.